public/app/plugins/datasource/cloud-monitoring/plugin.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "Stackdriver",
+  "name": "Cloud Monitoring",
   "type": "datasource",
-  "id": "stackdriver",
+  "id": "cloud-monitoring",
```

Comment:
It generates a lot of extra checks and handling changing the id of the plugin. Thinking about introducing a new property in plugin model json called `legacyId`or something that can hold "stackdriver" and be used in plugins package. Thoughts?